avtozip/templates/base.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <title>{% block title %}{% endblock %}</title>
+  {% block staticdata %}{% endblock %}
```

Comment:
rename block to `css`
add blocks `js_header` to the header
add block 'js_content' which should be placed at the end of body